subsys/net/lib/conn_mgr/Kconfig

```diff
@@ -35,4 +35,8 @@ config NET_CONNECTION_MANAGER_PRIORITY
 	help
```

Comment:
In the commit message `net: conn_mgr: Agnostic iface control multicasting`, the wording "multicast" is really confusing to me. I know IP multicasting but that is probably not the same thing? Could we change this to something less confusing.

Reply:
I've renamed the commit message to `Bulk iface commands`